RELEASE/scripts/autoscend/combat/auto_combat_default_stage5.ash

```diff
@@ -76,34 +75,34 @@ string auto_combatDefaultStage5(int round, monster enemy, string text)
 			return useSkill(stunner);
 		}
 
-		if(canUse($skill[Shoot Ghost], false) && (my_mp() > mp_cost($skill[Shoot Ghost])) && !contains_text(combatState, "shootghost3") && !contains_text(combatState, "trapghost"))
```

Comment:
See comment in Ed Combat

Reply:
refactored the entire proton pack combat code further